### ISSUE TITLE
Click-to-view-attribute is broken

### ISSUE THREAD

**Issue by kgjenkins** [cul-it/cugir-geoblacklight#3]
When viewing a single item (such as https://cugir.library.cornell.edu/catalog/8163 ), the user should be able to "click on map to inspect values", but it doesn't work.

Clicking the map attempts to send a POST request to https://cugir.library.cornell.edu/wms/handle but that URL is not set up on the server.  I'm not sure how GeoBlacklight is intended to handle these clicks, but in the end it should send a GET request to geoserver/wms?REQUEST=GetFeatureInfo...

**Comment by kgjenkins** [comment]
Huda wrote via e-mail:

> ...should be fixed by going to the most recent version of geoblacklight (which is 0.11) according to this https://github.com/geoblacklight/geoblacklight/issues/359 and also based on the fact that 0.11 on my machine works fine in this situation.
> 
> The error is that when you click on the preview map on https://cugir.library.cornell.edu/catalog/8163, the attributes should load properly and they are not currently loading on the VM (because of this faraday 0.9.2 error – I can send the log file if you’d like).
> 
> I tried to do a bundle update earlier but I may not have the proper permissions to do anything so Darcy may have to do that (and Darcy, you’re better at handling the intersection of rails, git, and permissions so I don’t want to mess this up).
> 
> Updating to the latest version of blacklight should fix this issue.

**Comment by dbranchini** [comment]
Can't be addressed until #18 is done.

**Comment by dbranchini** [comment]
I updated to geoblacklight 0.12 to fix this.  The 'data-layer-id' is working correctly now so I believe this will fix the issue, however, I can't fully test it until the geoblacklight metadata is updated and solr is re-indexed. The preview feature is using solr fields for the geoserver path/url and those urls/paths have been changed with the recent changes to geoserver.  I will look at this again once the solr is re-indexed to verify that it is working correctly.
